src/store/graphql/variants/models.ts

```diff
@@ -31,6 +35,11 @@ export type Frequencies = {
       ac: number;
     };
   };
+  topmed: { ac: number; af: number; an: number; homozygotes: number; heterozygotes: number };
+  one_thousand_genomes: { ac: number; af: number; an: number };
+  gnomad_exomes_2_1: { ac: number; af: number; an: number; homozygotes: number };
+  gnomad_genomes_2_1: { ac: number; af: number; an: number; homozygotes: number };
+  gnomad_genomes_3_0: { ac: number; af: number; an: number; homozygotes: number };
```

Comment:
does reuse type make sense for future refactoring and readability? what do you think?

```
  type FreqAll = { ac: number; af: number; an: number }
  type Freqgnomad = FreqAll & {  homozygotes: number };

...
  topmed: Freqgnomad & { heterozygotes: number };
   one_thousand_genomes: FreqAll ;
   gnomad_exomes_2_1: Freqgnomad;
   gnomad_genomes_2_1: Freqgnomad;
   gnomad_genomes_3_0: Freqgnomad;
```